pisa-proxy/parser/mysql/src/grammar.y

```diff
@@ -5988,6 +5989,31 @@ show_create_table_stmt -> Box<ShowCreateTable>:
     }
 ;
 
+show_keys_stmt -> Box<ShowKeysStmt>:
+    'SHOW' opt_extended keys_or_index from_table opt_db opt_where_clause
+    {
+    	Box::new(ShowKeysStmt {
+    	   span: $span,
+    	   opt_extended: $2,
+    	   keys_or_index: $3,
+    	   from_table: $4,
+    	   opt_db: $5,
+    	   opt_where_clause: $6,
+    	})
+    }
+;
+
+keys_or_index -> KeysOrIndex:
+       'INDEX'              { KeysOrIndex::Index }
+     | 'INDEXES'            { KeysOrIndex::Indexes }
+     | 'KEYS'               { KeysOrIndex::Keys }
+;
+
+opt_extended -> Option<String>:
+       /* empty */          { None }
+     | 'EXTENDED'           { Some(String::from("EXTENDED")) }
+;
+
```

Comment:
This should be  return `bool` type here, because  `String`  type need more memory allocation.
`
opt_extended -> bool:
       /* empty */          { false }
     | 'EXTENDED'           {  true }
;
`

Reply:
> bool

OK Thanks.